### DISCUSSION TITLE
Atualizei e não consigo restaurar meu backup

### DISCUSSION THREAD

**Discussion by Soldadodelcaro** [RamonSilva20/mapos#1977] [General]
Olá, desinstalei meu mapos para atualizar, quando reinstalei fui devolver meu backup e aparece um erro, antes de atualizar eu tinha personalizado ele apenas com minhas logos e mais nada, alguém pode me ajudar.. segue a imagem do erro.
![16690860348377732835941350935628](https://user-images.githubusercontent.com/118787155/203210702-c5023fdd-3578-417f-a06e-75cfdb31f132.jpg)

**Comment by marlontomaz** [comment]
Passei pelo mesmo problema na hora de atualizar da V2 para v4.... e a solução foi importar cada update de versão do sql

abra a pasta updates - > e importe um a um direto no banco de dados, isso vai aplicar a atualização dos novos campos que foram criados, no decorrer de cada versão do sistema.

**Comment by Soldadodelcaro** [comment]
Meu amigo, me desculpe a falta de conhecimento, você conseguiria me dar um
suporte para fazer isso? Qual valor você cobra pra me dar esse apoio?

Em ter, 22 de nov de 2022 08:49, Marlon Tomazeli ***@***.***>
escreveu:

**Comment by Soldadodelcaro** [comment]
Fiz isso, importei pelo Phpmyadmin um por um até a última versão aí importei meu backup por último e o erro permanece. Por favor me ajude. Obrigado aí irmão Em 22 de nov. de 2022 10:03 AM, Thiago Del Caro ***@***.***> escreveu:

**Comment by Soldadodelcaro** [comment]
Num último suspiro peguei a última atualização e importei agora depois de tudo e voltou... Só perdeu minhas logos, mas isso coloco agora... Muito obrigado meu amigo, salvou real... Manda o pix aí que vou lhe enviar umas Heinekens... Muito obrigado mesmo... Responderam rápido, vocês são feras.Em 22 de nov. de 2022 10:29 AM, Thiago Marins del caro ***@***.***> escreveu:Fiz isso, importei pelo Phpmyadmin um por um até a última versão aí importei meu backup por último e o erro permanece. Por favor me ajude. Obrigado aí irmão Em 22 de nov. de 2022 10:03 AM, Thiago Del Caro ***@***.***> escreveu:

**Reply by marlontomaz** [reply]
que bom que deu certo amigo, peço desculpas pela falta de retorno.
@marlontomazeli
